docs/source/troubleshooting.rst

```diff
@@ -0,0 +1,20 @@
+Troubleshooting
+===============
+
+- Job is slow or OOMs while using an aggregate like ``collect_list`` or
+  ``sample_call_summary_stats``
+
+  * Try disabling the `ObjectHashAggregate
+    <https://github.com/apache/spark/commit/27daf6bcde782ed3e0f0d951c90c8040fd47e985>`_ by setting
+    ``spark.sql.execution.useObjectHashAggregateExec`` to ``false``
+
+- Job is slow or OOMs while writing to partitioned table
```

Comment:
nit: the line spacing is awkward on rendering. The solution is closer to the next item than the item.

Reply:
Fixed

Reply:
Fixed by the css change

Reply:
Nice